packages/create-app/src/prompts/promptApplicationUrl.ts

```diff
@@ -33,8 +33,12 @@ export async function promptApplicationUrl(
       {
         type: "select",
         options: [
-          { label: "Yes, prefill it for me", value: "yes" },
-          { label: "No, I will fill it in myself later", value: "no" },
+          { label: "Yes, let me fill it here", value: "yes" },
+          {
+            label:
+              "No, I will set `VITE_FOUNDRY_REDIRECT_URL` variable in `.env.production` file later",
+            value: "no",
+          },
```

Comment:
For the no label I would be more hesitant on referencing the exact place to change because that depends on the template (for example nextjs uses a `NEXT_PUBLIC_` prefix). The READMEs that are generated for the template do [explain this](https://github.com/palantir/osdk-ts/blob/3cfe65fb7a795cbe35f4a4d51fc51ccfadbe32e7/packages/create-app/templates/template-react/README.md.hbs#L29-L31) and we could technically make this label aware of the template that was chosen as well but I think it would be enough to defer to the README.

```suggestion
          { label: "Yes, let me fill it here", value: "yes" },
          { label: "No, I will fill it in myself later", value: "no" },
```

Reply:
@tzyl How about just referencing the name of the variable?

Reply:
The static variable name here is still incorrect as it would need to switch depending on the template selected in the previous prompt. Another suggestion is after the user selected "no" to put another line with the extra detail telling them to follow the generated README instructions later:

```
✔ Do you know the URL your production application will be hosted on? This is required to create a production build of your application with the correct OAuth redirect URL.
No, I will fill it in myself later

You have chosen not to fill in the URL your production application will be hosted on. Follow the instructions in the generated project README once you are ready to create a production build of your application.
```

Reply:
Ended up suggesting 
`No, I will set it up later based on the instructions in the README`
One line is simple